docs/guides/building-forms.md

```diff
@@ -395,6 +395,6 @@ That's about it. After reading this guide, here's what you'll learn:
 - Validating forms data with and without JavaScript
 - Using Next.js Sereverless Functions to handle `req` and `res` from the client and server
 
-Next.js and Vercel provide you with an all-in solution for hosting web forms and handling submissions with serverless functions. Use Next.js as the client for your JavaScript-based interaction and Vercel for its powerful serverless functions to handle the server.
+Next.js provide you with an all-in solution for hosting web forms and handling submissions with serverless functions. Use Next.js as the client for your JavaScript-based interaction and Vercel for its powerful serverless functions to handle the server.
```

Comment:
Could we remove Vercel from this guide entirely? 🙏 (and serverless here)

Reply:
API Routes would be the proper term to reflect this. Self-hosted, API routes can run on your Node.js server. Deployed to Vercel, they are serverless functions 👍

Reply:
Noted. 👍